### PR TITLE
Remove Laminas JSON library

### LINKED ISSUE #18412: Replace laminas/json
Package is abandonned: https://github.com/laminas/technical-steering-committee/blob/main/meetings/minutes/2023-12-04-TSC-Minutes.md

Candidate for replacement:

- https://packagist.org/packages/balping/json-raw-encoder

### PR DESCRIPTION
**Pull request by cconard96**
## Checklist before requesting a review

- [x] I have read the CONTRIBUTING document.
- [x] I have performed a self-review of my code.

## Description

Fixes #18412

This library is abandoned and wasn't strictly needed.

### CODE REVIEW

**Review by cedric-anne — APPROVED** [review]
Seems technically OK, not tested manually.

**Review by AdrienClairembault — APPROVED** [review]

**Review by orthagh — CHANGES_REQUESTED** [review]
Just after checking out this PR, some things are broken:

- labels of horizontal tabs widget  show Nan or nothing
- Point markers of line/area charts are not displayed anymore when you enable the option
- edit an option of any cards, remove the controls of the cards after saving:
image

Not related to this PR:
- There is a "Show legend" option that doesn't seem to be working anymore

**Comment by cconard96** [comment]
>     * edit an option of any cards, remove the controls of the cards after saving:

Also an issue on main

File: src/Glpi/Dashboard/Widget.php [hunk 1]
```diff
@@ -1543,39 +1517,44 @@ function(param) {
...
+        $js = TemplateRenderer::getInstance()->renderFromStringTemplate(<<<TWIG
+            <script type="module">
+                const target = GLPI.Dashboard.getActiveDashboard() ?
+                    GLPI.Dashboard.getActiveDashboard().element.find('#{{ chart_id }} .chart')
+                    : $('#{{ chart_id }} .chart');
+                const chart_options = {{ options|json_encode|raw }};
+                
+                $.each(chart_options.series, function (index, serie) {
+                    if ({{ show_points ? 'true' : 'false' }}) {
+                        serie['symbol'] = (param) => param > 0 ? 'circle': 'none';
```
Comment (cedric-anne) [inline src/Glpi/Dashboard/Widget.php:1536]: Variable name was `value` in the previous code. I guess it matches the component documentation.
```suggestion
                        serie['symbol'] = (value) => value > 0 ? 'circle': 'none';
```

**Comment by orthagh** [comment]
In Current state, a line/area graph (ex "ticket status by month") give me a js error: "Uncaught TypeError: serie.label is undefined"

**Review by orthagh — APPROVED** [review]
I don't see so far any issues related to the pr itself.

I found one valid on main also:

- legend option -> doesn't work anymore
- some options of a card are not saved (some are working like gradient, chart type, etc):
  - palette
  - labels
  - legend

I'll create an issue for that

**Comment by trasher** [comment]
Could you please rebase to solve conflicts?

**Pull request merged**